cloudinit/config/cc_disk_setup.py

```diff
@@ -605,8 +613,8 @@ def get_partition_mbr_layout(size, layout):
     """
 
     if not isinstance(layout, list) and isinstance(layout, bool):
-        # Create a single partition
-        return "0,"
+        # Create a single partition, default to Linux
+        return ",,83"
```

Comment:
I'd added the "83" to align/be consistent with the existing reference on line 631 (to show that "layout: true" and "layout: [100]" are equivalent).

I can change this but it would make sense for the 2 places to be consistent.